staff/sys/stdin2syslog

```diff
@@ -1,49 +1,22 @@
-#!/usr/bin/env python3
-"""Write stdin logs to OCF's remote syslog server.
+#!/bin/bash
+set -euo pipefail
 
-You can use arbitrary stream names (without configuring them before); each
-stream will show up at syslog:/var/log/remote/<stream>.log.
-"""
-import argparse
-import re
-import subprocess
-import sys
-
-
-VALID_STREAM_NAME = re.compile('^[a-z0-9\-_]+$')
-
-
-class LoggingFailure(Exception):
-    pass
+if [ $# -lt 1 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+    echo "Usage: $0 stream
 
+Write stdin logs to OCF's remote syslog server.
 
-def write_line(stream, line):
-    """Write a line to a stream.
-
-    This might make sense to move to ocflib some day.
-    """
-    if not VALID_STREAM_NAME.match(stream):
-        raise ValueError('Bad stream name: {}'.format(stream))
-
-    ret = subprocess.call((
-        'logger', '-d', '-n', 'syslog', '-P', '514', '-t', stream, '--', line,
-    ))
-    if ret != 0:
-        raise LoggingFailure('logger exited nonzero: {}'.format(ret))
-
-
-def main(argv=None):
-    parser = argparse.ArgumentParser(
-        description=__doc__,
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-    )
-    parser.add_argument('stream')
-    args = parser.parse_args(argv)
+You can use arbitrary stream names (without configuring them before); each
+stream will show up at syslog:/var/log/remote/<stream>.log.
+"
```

Comment:
Not a huge fan of the indentation here, but not sure what would be better, maybe a here doc or something?

Reply:
Usually you would either save it to a variable at top level indentation or just use multiple `echo` commands.